snapshots/README.md

```diff
@@ -0,0 +1,48 @@
+# Snapshot
+In this demo we will show how to define and configure a Snapshot Trigger for a given Aggregate.
+
+Eventually you will need to speed up the Event Sourcing part of your Aggregate because you already have too many events. That is when a Snapshot is desirable. Axon Framework helps you on that by simply configuring a `SnapshotTriggerDefinition` and instructing your `Aggregate` to use that.
```

Comment:
```suggestion
Eventually you will need to speed up the Event Sourcing part of your Aggregate because the number of events exceeds your defined aggregate loading time. That is when a Snapshot is desirable. Axon Framework helps you on that by simply configuring a `SnapshotTriggerDefinition` and instructing your `Aggregate` to use that.
```